tests/PhpGeneratorTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+ob_start();
+require __DIR__ . '/../jkm.php';
+ob_end_clean();
+
+$data = json_decode(file_get_contents(__DIR__ . '/../data.json'), true);
+$introductions = $data['introductions'];
+$whos = $data['who'];
+$hows = $data['how'];
+$whats = $data['what'];
+$whys = $data['why'];
+$culminations = $data['culminations'];
+
+$statements = [];
+foreach ($introductions as $introduction) {
+    foreach ($whos as $who) {
+        foreach ($hows as $how) {
+            foreach ($whats as $what) {
+                foreach ($whys as $why) {
+                    foreach ($culminations as $culmination) {
+                        $statements[] = $introduction . $who . $how . $what . $why . $culmination;
+                    }
+                }
+            }
+        }
+    }
+}
+
+for ($i = 0; $i < 100; $i++) {
+    assert(in_array(generate(), $statements));
+}
+
+for ($i = 0; $i < 20; $i++) {
+    $statement = generate();
```

Comment:
hmm, a co jeśli byśmy użyli 
```suggestion
    $output=null;
    $retval=null;
    exec('php ../jkm.php', $output, $retval);
    
    tym sposobem byśmy mogli sprawdzic rzeczywiste odpalenie skryptu z $output porównywać wynik, jesli mamy dostęp do php w shell, inaczej mozna obudowac to w docker i tam juz mozna bedzie otestowac reszte języków. Co o tym myślisz?
```

Reply:
👌 git, dobry pomysł